README.md

```diff
@@ -32,159 +30,89 @@ A version of the Swagger Petstore API is displayed by default.
-  </body>
-</html>
-
 ```
 
-For step-by-step instructions for how to get started using Redoc
-to render your OpenAPI definition, refer to the
-[**Redoc quickstart guide**](https://redocly.com/docs/redoc/quickstart/) and [**How to use the HTML element**](https://redocly.com/docs/redoc/deployment/html/).
+Open the HTML file in your browser, and your API documentation is shown on the page.
+
+Add your own `spec-url` to the `<redoc>` tag, this attribute can also be a local file. The JavaScript library can also be installed locally using `npm` and served from your own server, see the [HTML deployment documentation](https://redocly.com/docs/redoc/deployment/html/) for more details.
+
+### More usage options
+
+Check out the [deployment documentation](./deploment/index/md) for more options, and detailed documentation for each.
+
+## Redoc vs. Reference
+
+Redoc is Redocly's community-edition product. Looking for something more?
+We also offer [hosted API reference documentation](https://redocly.com/docs/api-registry/guides/api-registry-quickstart/)
+with additional features including:
+
+* Try-it console
+* Automated code samples
+* Pagination
+* Extra theme options
+
+### Documentation and resources
+
+- [Reference docs](https://redocly.com/docs/api-reference-docs/getting-started/) - we take care of the hosting
+- [Redoc](https://redocly.com/docs/redoc/) - detailed documentation for this open source project (also in the `docs/` folder)
+- [Command-line interface to bundle your docs into a web-ready HTML file](https://redocly.com/docs/cli/commands/build-docs/)
+- API linting, bundling, and much more with open source [Redocly CLI](https://redocly.com/docs/cli)
+
+## Showcase
+
+A sample of the organisations using Redocly tools in the wild:
+
+- [Rebilly](https://api-reference.rebilly.com/)
+- [Docker Engine](https://docs.docker.com/engine/api/v1.25/)
+- [Zuora](https://www.zuora.com/developer/api-reference/)
+- [Discourse](http://docs.discourse.org)
+- [Commbox](https://www.commbox.io/api/)
+- [APIs.guru](https://apis.guru/api-doc/)
+- [BoxKnight](https://www.docs.boxknight.com/)
 
+_Pull requests to add your own API page to the list are welcome_
 
 ## Configuration
 
-### Security Definition location
-You can inject the Security Definitions widget into any place in your definition `description`.
-For more information, refer to [Security definitions injection](docs/security-definitions-injection.md).
+Redoc is highly configurable, see the [configuration documentation](docs/config.md) for details.
 
 ### OpenAPI specification extensions
 Redoc uses the following [specification extensions](https://redocly.com/docs/api-reference-docs/spec-extensions/):
+
 * [`x-logo`](docs/redoc-vendor-extensions.md#x-logo) - is used to specify API logo
 * [`x-traitTag`](docs/redoc-vendor-extensions.md#x-traitTag) - useful for handling out common things like Pagination, Rate-Limits, etc
```

Comment:
I'm not sure about this wording here - "handling out" - not sure what it means

Reply:
I updated it, it's basically to revert the tags to being used as just tags, not as navigation.